Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/defender.yml

```
@@ -1,12 +1,12 @@
-﻿- type: entity
+- type: entity
```

I think your editor changed the encoding of the file accidentally judging by the modified first line
```suggestion
- type: entity
```

---

don't know how i can fix it, doesn't seem to actually change anything, might just be git being git, should be good tho